app/src/organisms/ProtocolSetupParameters/index.tsx

```diff
@@ -84,6 +86,14 @@ export function ProtocolSetupParameters({
           ({ ...parameter, value: parameter.default } as ValueRunTimeParameter)
     )
   )
+  const hasMissingFileParam =
+    runTimeParametersOverrides?.some(
+      parameter =>
+        parameter.type === 'csv_file' &&
+        parameter.file?.id == null &&
+        parameter.file?.file == null &&
+        parameter.file?.filePath == null
```

Comment:
I might be wrong but from yesterday's conv

the condition would be

```suggestion
        parameter.type === 'csv_file' &&
        ((parameter.file?.id == null &&
        parameter.file?.file == null) || parameter.file?.filePath == null)
```

Reply:
correct, thank you for the catch!